proposals/vip_12.ts

```diff
@@ -0,0 +1,17 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const vip_x: ProposalDescription = {
+  title: 'VIP-12: Add new PSMs to PCV Guardian Whitelist',
+  commands: [
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'addWhitelistAddresses(address[])',
+      arguments: [['{voltV2DaiPriceBoundPSM}', '{voltV2UsdcPriceBoundPSM}']],
+      description: 'Add new DAI, and USDC PSMs to PCV Guardian whitelist'
```

Comment:
This proposal also needs to handle adding these PSM's to the ERC20 allocator, and removing the two previous PSM's from the allocator as well as connecting the PCV Deposits to the newly connected PSMs.